src/olympia/migrations/938-drop-l10n-from-bio-user-profile.sql

```diff
@@ -0,0 +1,23 @@
+/* FOREIGN KEY is prod's - you will need to change this for your local DB */
+
+/* Add the new column and drop the constraint so we can clear the translations.*/
+ALTER TABLE `users`
+    ADD COLUMN `_bio` longtext NULL,
+    DROP FOREIGN KEY `users_ibfk_1`;
+
+/* Copy across the translated values.  There are some users that have multiple
+localizations so we're going to use the default locale (lang) localization. */
+UPDATE `users`
+    SET `_bio` = (
+        SELECT `localized_string` FROM `translations`
+        WHERE `id`=`bio` AND `locale`=`lang`);
+
+/* Clear out the now obsolete translations. */
+DELETE FROM `translations` WHERE `id` in (SELECT `bio` FROM `users`);
+
+/* Drop the old columns and rename bio_ to take the place of bio. */
+ALTER TABLE `users`
+    DROP COLUMN `bio`,
+    DROP COLUMN lang,
+    DROP COLUMN region,
+    CHANGE COLUMN `_bio` `bio` longtext NULL;
```

Comment:
Alternatively:
1. add the new `_bio` column as nullable text; copy across text; *remove* field in model temporarily (and all references in existing code)
2. rename column to `bio`, add back the field with the new name in model

Or:
1. add new `biography` column; copy across text; (change column in model to be `TextField` and refer to/be called `biography`.)
2. drop `bio` column

Note: in any case, make sure the old column is nullable.